Readme.md

```diff
@@ -192,18 +194,18 @@ There are several properties exposed from the hook to deal with the state throug
 | error            | any error thrown within the handleSubmit function is stored here as an error object                    |
 | submissionStatus | Displays the current status of the submission process                                                  |
 
-#### Submission Status
+### Submission Status
```

Comment:
quickfix to have some space between the title and the table above it. if we want to stick to h4 then we can change the css file (not sure if it's also generated)